config/config.yml.dist

```diff
@@ -1,8 +1,15 @@
-CleanBranch:
+MySQL:
+  hostname: ${db_host}
+  username: ${db_username}
+  password: ${db_password}
+
+Jira:
   hostname: ${host}
```

Comment:
Can you rename this as `jira_host` as well?
So if you run `vendor/bin/tempa` the key for replacement is easier to distinquish